RELEASE_NOTES.rst

```diff
@@ -1,3 +1,93 @@
+coala 0.10 - PolarBear
+=======================
+
+For this release we had 168 commits contributed by 66 unique contributors
+over 2.5 months
+
+The name of this release is PolarBear to reflect changes we have made to our
+release process. We have implemented a release freeze before all releases
+to serve as a testing period for all staged changes. This should also help
+us keep releases on schedule.
+
+This release is an enhancement release which brings the usual slew of
+improvements to documentation and api. We have started preparing the codebase
+for the big changes that are outlined in the cEPs like section inheritance
+and the next generation core.
+
+**General**
+
+- Test cases have been improved across the board so they catch more errors
+  before production.
+- Log output via json has been added which is accessible by using the
+  ``--log-json`` option.
+- coala domain has been changed from ``coala-analyser.org`` to ``coala.io``.
+  Please file a bug if you find any broken links or instances of the old url.
+
+**Usability**
+
+- Ignore statements in multi-line comments should be supported properly now.
+- emacs has been added to list of editors that coala supports. coala will
```

Comment:
Isn't "coala will now output a warning if you specify an editor that has no configuration" another item?